examples/CMakeLists.txt

```diff
@@ -1,5 +1,5 @@
 include_directories(${CMAKE_SOURCE_DIR} ${Boost_INCLUDE_DIRS} ${Libmsgpack_INCLUDE_DIRS})
-link_libraries(${Boost_LIBRARIES} ${CMAKE_THREAD_LIBS_INIT})
+link_libraries(${Boost_LIBRARIES} ${CMAKE_THREAD_LIBS_INIT} crypto)
```

Comment:
`crypto` seems like it could go into the target_link_libraries() list for only the wampcra example, not necessarily all of them. I also wonder how safe it is to use `crypto` as a dependency without checking for it's existence. (Does that work on all platforms?)